norent/scaffolding.py

```diff
@@ -9,7 +9,7 @@
 
 
 # This should change whenever our scaffolding model's fields change.
-VERSION = "1"
+VERSION = "2"
```

Comment:
Have you tried _not_ changing this?  Because of how the norent scaffolding uses `pydantic.BaseModel` (unlike the onboarding scaffolding, which just stores raw dictionaries), I think it's possible that you might be able to get away with not changing this (and thereby not disrupting anyone who's currently onboarding) since `preferred_first_name` would just default to the empty string on the server side.  See my other comment about the onboarding `FIELD_SCHEMA_VERSION` for more details on why it'd be preferable to not have to increment this if possible.